go.mod

```diff
@@ -12,6 +12,8 @@ require (
 	github.com/heimweh/go-pagerduty v0.0.0-20230908205257-a96e6c05f7ea
 )
 
+replace github.com/heimweh/go-pagerduty v0.0.0-20230908205257-a96e6c05f7ea => github.com/swbradshaw/go-pagerduty v0.0.0
```

Comment:
Annotate: Temporary replace version with tagged version that supports custom fields